### PR TITLE
Add basic movement

### PR DESCRIPTION
**Pull request by toberge**
- Horizontal strafing and jumping.
- Ground detection with Physics.BoxCast, visualized with a gizmo.
- Strafing is less impactful in the air.
- Releasing movement input when grounded causes the player to be pushed back slightly, effectively dampening their movement and making it closer to an abrupt stop.
- Ignore blob files that appear when Unity/Mono crashes on my PC :upside_down_face:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -54,6 +54,7 @@ ExportedObj/
 
 # Unity3D generated file on crash reports
 sysinfo.txt
+*.blob
```
[the remark this answers is not in the document]
Comment (toberge) [reply .gitignore:57]: Good point :+1:

[the remark this answers is not in the document]
Comment (ddabble) [reply .gitignore]: Ooh, nice comments, now! 😄

**Comment by ddabble** [comment]
I'd say keep the TODO comments, as they can definitely be useful for future development! But maybe make them more clear/verbose, so that they convey their intent better 😊

**Review by Fueredoriku — APPROVED** [review]
Looks good! 😎👍

We should probably have a max velocity/magnitude, but I would say that isn't strictly necessary for this PR. (One has to know how to abuse it to do so)

To illustrate: first attempt is normal jumping from slope, while second is spamming jump button on slope: 
highJump

We can alternatively implement this later while tuning movement when we have a first-person perspective from player, as that might arguably be better from a devlopment perspective.

Regardless I approve of this PR in it's current form, good work!

**Pull request merged**